cnd/src/swap_protocols.rs

```diff
@@ -84,3 +85,28 @@ pub trait RefundAction {
 
     fn refund_action(&self) -> Option<Self::Output>;
 }
+
+#[derive(Clone, Debug)]
+pub struct HanEthereumEther {
```

Comment:
We already have modules for protocols, can we move these types into there and shorten their names by importing them with more paths?

Reply:
Good idea, will do.